src/edatetime.erl

```diff
@@ -18,13 +18,13 @@
 -export([iso8601/1, iso8601_basic/1]).
 -export([tomorrow/1, yesterday/1]).
 
--type timestamp() :: pos_integer().
+-type timestamp() :: integer().
```

Comment:
Surely, there must have been something there _before_ the Unix epoch began. :)
I've run into this issue when expressing user's birthdays as timestamps, to give an example.